src/EditorFeatures/CSharpTest/SymbolKey/SymbolKeyCompilationsTests.cs

```diff
@@ -269,8 +267,6 @@ public enum EGoo { Zero, One }
 
             var src2 = @"using System;
 
-public delegate void DGoo(int p1, string p2); // add 1 more parameter
```

Comment:
intentional change.  we had a test that said the old dleegate should resolve to the new one.  but we do not want thsi behavior at all.  these are distinct types